cartridge-cli.lua

```diff
@@ -2925,6 +2961,7 @@ local function pack_rpm(opts)
 
     body = lead .. buf_pad_to_8_byte_boundary(signature_header) .. body
 
+    info('Write RPM file')
```

Comment:
I'm not sure that user wants to see such details, let's move it to verbose

Reply:
It confuses a little when you see smth like `Pack in CPIO` and then nothing happens for a few minutes.
I think that messages like this 

```bash
Write application tmpfiles configuration
Create CPIO archive
163006 blocks
Compress it using GZIP
# here nothing happens for a few minutes
```
makes packing process more clear.

Reply:
I sure that we need more log levels, but it should be performed in the separated MR.

Reply:
Ok